src/ontodia/data/sparql/sparqlDataProviderSettings.ts

```diff
@@ -0,0 +1,260 @@
+// this is dataset-schema specific settings
+export interface SparqlDataProviderSettings {
+    // default prefix to be used in every query
+    defaultPrefix: string;
+    // property to use as label in schema (classes, properties)
+    schemaLabelProperty: string;
+
+    // property to use as instance label todo: make it an array
+    dataLabelProperty: string;
+
+    // full-text search settings
+    ftsSettings: FullTextSearchSettings;
```

Comment:
Try to avoid abbreviations in property names, e.g. this property could be named just `fullTextSearch`. No need to append `settings` at the end.